src/sonic-frr/patch/0012-Ensure-ospf_apiclient_lsa_originate-cannot-accidently-write-into-stack.patch

```diff
@@ -0,0 +1,108 @@
+From d8d77d3733bc299ed5dd7b44c4d464ba2bfed288 Mon Sep 17 00:00:00 2001
+From: Donald Sharp <sharpd@nvidia.com>
+Date: Wed, 20 Jul 2022 16:43:17 -0400
+Subject: [PATCH 1/3] ospfclient: Ensure ospf_apiclient_lsa_originate cannot
```

Comment:
>PATCH 1/3 [](http://example.com/codeflow?start=10&length=9)

I see "PATCH 1/3" is backport of frr upstream commit. How about others? Could you explain 2/3 and 3/3 in PR description?